.cirrus.yml

```diff
@@ -128,7 +141,13 @@ task:
         matrix:
           CHANNEL: "master"
           CHANNEL: "stable"
-      script:
+      pathified_unit_test_script:
+        # Run tests with path-based dependencies to ensure that publishing
+        # the changes won't break tests of other packages in the respository
+        # that depend on it.
+        - ./script/tool_runner.sh make-deps-path-based --target-dependencies-with-non-breaking-updates
+        - dart pub global run flutter_plugin_tools test --run-on-dirty-packages --exclude=flutter_image
```

Comment:
Rather than duplicate the `flutter_image` exclusion, we should switch both lines to a `script/configs/dart_unit_tests_exceptions.yaml`-based exclusion, so we don't end up with things getting out of sync. (See the `readme-check` section above for an example of this.)